hasjob/templates/layout.html.jinja2

```diff
@@ -217,7 +217,6 @@
       {%- endraw %}
     </div>
   {% endmacro %}
-  {% if not g.kiosk %}
```

Comment:
Login and other user buttons features should be disabled in kiosk mode, since it is meant to be an anonymous mode. Otherwise good to merge.